### DISCUSSION TITLE
user interface

### DISCUSSION THREAD

**Discussion by jhls109** [aaronwmorris/indi-allsky#1502] [General]
Hi Aaron, I also posted this on cloudynights (on the long indi-allsky thread) with a picture I referenced below. 

I've been running Indi-Allsky since July. I only put the camera out on nights when I'm imaging, which lately has been more often here New England due to much better weather and not much wildfire smoke. It did take me a while to get my camera and Indi-Allsky setup the way I like, but now it's running great, and providing nice images! I was able to capture an aurora in mid-August, that was really cool! 
 
I am an RPI and Indi-Allsky novice, so I'd appreciate some help or comments on the following:
- Separate topic regarding the Indi-Allsky interface. On the main "home" page, when I click on media, then timelapse, the page opens to selections for keogram, star trail, Timelapse etc., as shown here:

Everything works as expected, except if I click on the image on the right, which is a keogram, that opens the timelapse!

Thanks in advance for your help!
John

**Comment by aaronwmorris** [comment]
Hey, thank you for the feedback.  What you are seeing is the intended design.  The keogram kind of represents the entire day or night period and clicking on it will open the timelapse video for the represented time period.

There is a separate link on the left hand side labeled "Keogram" which will open the actual keogram image.  (The image on the right is technically a scaled thumbnail, to decrease page load times.)
